dev/provider_packages/SETUP_TEMPLATE.cfg.jinja2

```diff
@@ -23,19 +23,54 @@
 
 
 [metadata]
-name = Airflow {{ PROVIDER_TYPE }}: {{ PACKAGE_PIP_NAME }}
+name = {{ PACKAGE_PIP_NAME }}
 summary = {{ PROVIDER_TYPE }} for Apache Airflow. Implements {{ PACKAGE_PIP_NAME }} package
-description_file = README.md
-author = Apache Airflow PMC
+description = {{ PROVIDER_TYPE }} package {{ PACKAGE_PIP_NAME }} for Apache Airflow
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Apache Software Foundation
 author_email = dev@airflow.apache.org
-license = Apache License, Version 2.0
+url = https://airflow.apache.org/
+download_url = https://archive.apache.org/dist/airflow/{{ PROVIDERS_FOLDER }}
+license = Apache License 2.0
 license_files =
    LICENSE
    NOTICE
+classifiers =
+   Development Status :: 5 - Production/Stable
+   Environment :: Console
+   Environment :: Web Environment
+   Intended Audience :: Developers
+   Intended Audience :: System Administrators
+   Framework :: Apache Airflow
```

Comment:
I am not 100% sure if we want to add also "Framework :: Apache Airflow" as classifier here. I guess we do (otherwise only "apache-airlfow" package would be applicable to have it) @ashb @kaxil WDYT?

Reply:
Yup, agreed, we should add it. 

I consider it as "Projects using Apache Airflow"

Reply:
Cool!